servlet/src/main/java/io/undertow/servlet/core/ServletExtensionHolder.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2014 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package io.undertow.servlet.core;
+
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+
+import io.undertow.servlet.ServletExtension;
+
+public class ServletExtensionHolder {
+
+    private static List<ServletExtension> extensions = new CopyOnWriteArrayList<>();
```

Comment:
Why have you added this? Undertow does not really use any static config, all config is passed in at deployment time.